plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/UnsupportedGradleTest.kt

```diff
@@ -14,7 +15,11 @@ class UnsupportedGradleTest : AbstractPluginTest() {
     @Test
     @DisabledOnOs(OS.WINDOWS)
     internal fun errorOnOldGradleVersion() {
-        project(GradleVersion.version("6.9.2")) {
+        if (getMajorJavaVersion() >= 21) {
+            // Gradle 8.5 needed for Java 21 anyway
```

Comment:
I'm confused by this as the next line isn't about Gradle 8.5

Reply:
I'll update this comment with a better explanation